pages/pipelines/dependencies.md

```diff
@@ -16,7 +16,7 @@ A wait step, as in the example below, is dependent on all previous steps complet
 steps:
   - command: "one.sh"
   - command: "two.sh"
-  - wait
+  - wait:
```

Comment:
I wonder if `wait: ~` is clearer than `wait:`.

I think the string "wait" is pretty clear:

```yaml
- command: "two.sh"
- wait
- command: "three.sh"
```

… but as @tessereth mentions it's harder/non-obvious to extend.

I worry that `- wait:` as an object _without any keys_ and without the `~` explicit null is a bit harder / less clear to read… the `:` seems to imply that whatever is next is part of the `wait`, but the `- command …` that is next isn't part of the wait.

```yaml
- command: "two.sh"
- wait:
- command: "three.sh"
```

So perhaps this is clearer in that particular scenario?

```yaml
- command: "two.sh"
- wait: ~
- command: "three.sh"
```

Reply:
I feel the docs should reflect best practices that readers should follow, as well as reducing any ambiguity that causes readers to think 'am I doing the right thing' (with this syntax)?

Therefore, I'm happy to take your guidance on this, and I'll amend this PR accordingly...